### PR TITLE
TELCODOCS-1558 Adding RN known and fixed issues section for telco core

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1558]: Adding RN known and bug fixes section for telco core

Version(s): 4.15 RN

Issue: :https://issues.redhat.com/browse/TELCODOCS-1558 and https://issues.redhat.com/browse/TELCODOCS-1557

Link to docs preview: https://71303--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1896,8 +1908,27 @@ In the following tables, features are marked with the following statuses:
...
 
+* In {product-title} {product-version}, all nodes use Linux control group version 2 (cgroup v2) for internal resource management in alignment with the default {op-system-base} 9 configuration. However, if you apply a performance profile in your cluster, the low-latency tuning features associated with the performance profile do not support cgroup v2.
++
+As a result, if you apply a performance profile, all nodes in the cluster reboot to switch back to the cgroup v1 configuration. This reboot includes control plane nodes and worker nodes that were not targeted by the performance profile.
++
+To revert all nodes in the cluster to the cgroups v2 configuration, you must edit the `Node` resource. For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2[Configuring Linux cgroup v2]. You cannot revert the cluster to the cgroups v2 configuration by removing the last performance profile. (link:https://issues.redhat.com/browse/OCPBUGS-16976[*OCPBUGS-16976*])
```
Comment (lpettyjo) [inline release_notes/ocp-4-15-release-notes.adoc:1915]: Might want to move that last sentence to be before the "more info" statement. It seems fairly important and you dont' want its message to be lost.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1896,8 +1908,27 @@ In the following tables, features are marked with the following statuses:
...
++
+This issue affects CPU load balancing features because these features depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods might run with a reduced number of CPUs. As a workaround, schedule a pod with a `best-effort` CPU Manager policy on the affected node. This pod will be the last admitted pod and this ensures the resources will be correctly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
+
+* When a node reboot occurs all pods are restarted in a random order. In this scenario it is possible that `tuned` pod started after the workload pods. This means the workload pods start with partial tuning which can affect performance or even cause the workload to crash. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
```
Comment (lpettyjo) [inline release_notes/ocp-4-15-release-notes.adoc:1929]: Add comma before "which". Can you use a more specific term instead of "crash"? Perhaps "fail".

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-15-release-notes.adoc:1915]: I am going to keep this in line  with 4.14 as it is carried forward from there.

**Comment by ffromani** [comment]
LGTM

**Comment by yanirq** [comment]
LGTM

**Pull request merged**